dags/create_final_actors.py

```diff
@@ -49,6 +49,20 @@ def apply_corrections_acteur(**kwargs):
         lambda x: shortuuid.uuid(name=x)
     )
 
+    # failed is identifiant unique is not unique in the dataframe df_acteur_merged
+    if not df_acteur_merged["identifiant_unique"].is_unique:
```

Comment:
Tu fais 2 fois le travail:
 - d'abord avec `is_unique` ([propriété qui engendre un traitement de la df](https://github.com/pandas-dev/pandas/blob/v2.2.3/pandas/core/base.py#L1069))
 - puis avec `df_duplicates = ... .duplicated(keep=False)` qui refait le taf sur la df

Tu pourrais faire 1 fois avec:
 - `df_duplicates = ... .duplicated(keep=False)`
 - `if not df_duplicates.empty:`